packages/providers/src/provider.ts

```diff
@@ -218,6 +220,7 @@ export type ProviderOptions = {
   ) => Promise<Response>;
   timeout?: number;
   cacheUtxo?: number;
+  retryOptions?: RetryOptions;
```

Comment:
I just noticed we forgot to add docs around this.

It's a matter of wrapping some tests in code regions and mentioning them in the guide.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1474/commits/42aa40ef433b7a0eb524063ceb4befc0bd8b497e